src/transform/FooterContainer.js

```diff
@@ -22,11 +22,12 @@ const updateBottomPaddingToAllowReadMoreToScrollToTop = window => {
  * @return {void}
  */
 const updateLeftAndRightMargin = (margin, document) => {
-  const elements = Polyfill.querySelectorAll(document, `
-    #pagelib_footer_container_menu_heading,
-    #pagelib_footer_container_readmore,
-    #pagelib_footer_container_legal
-  `)
+  const selectors = [
```

Comment:
Is this fixing some weird platform bug or is it a purely nonfunctional clean-up change / chore?